app/controllers/WhatDoYouWantToDoController.scala

```diff
@@ -58,29 +58,7 @@ trait WhatDoYouWantToDoController extends RasController with PageFlowController
     implicit request =>
       isAuthorised.flatMap {
         case Right(userId) =>
-          shortLivedCache.isFileInProgress(userId).flatMap {
-            case true =>
-              shortLivedCache.fetchFileSession(userId).map {
-                case Some(fileSession) =>
-                  fileSession.uploadTimeStamp match {
-                    case Some(timestamp) =>
-                      fileSession.userFile match {
-                        case Some(callbackData) =>
-                          Ok(views.html.what_do_you_want_to_do(whatDoYouWantToDoForm,fileIsInProgress, callbackData.fileId, readyForDownload))
-                        case _ =>
-                          Ok(views.html.what_do_you_want_to_do(whatDoYouWantToDoForm,fileIsInProgress, "", notReadyForDownload))
-                      }
-                    case _ =>
-                      Logger.error("[WhatDoYouWantToDoController][get] no timestamp retrieved")
-                      Redirect(routes.ErrorController.renderGlobalErrorPage)
-                  }
-                case _ =>
-                  Logger.error("[WhatDoYouWantToDoController][get] failed to retrieve file session")
-                  Redirect(routes.ErrorController.renderGlobalErrorPage)
-              }
-            case _ =>
-              Future.successful(Ok(views.html.what_do_you_want_to_do(whatDoYouWantToDoForm,noFileInProgress,"",noFileInProgress)))
-          }
```

Comment:
@eulogiogutierrez is there any functionality replacing this code snippet, I see this code snippet removed but cannot understand how this has been replaced or how this is related to the story thats in backlog?

Reply:
@raghu1978 - all of that was unnecessary (and some of it untested) code that was a remanent from when the whatdoyouwanttodo page did things it doesnt do anymore. All cases took to the same view but with different parameters that were, anyway, never used. Refactored for a cleaner solution as I would be testing that page intensively as part of this ticket anyway.